specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json

```diff
@@ -1604,6 +1604,19 @@
           "type": "string",
           "description": "The Uri of KeyVault.",
           "x-ms-client-name": "KeyVaultUri"
+        },
```

Comment:
Please add example for it, include create/update/get/list account.

Reply:
yes. example is required.

---
In reply to: [388050640](https://github.com/Azure/azure-rest-api-specs/pull/8585#discussion_r388050640) [](ancestors = 388050640)

Reply:
Once examples are added, I will merge this PR.

Reply:
Added example

Reply:
@tonykjose Please fix the CI errors. PrettierCheck can be fixed by adding the word into https://github.com/Azure/azure-rest-api-specs/blob/master/custom-words.txt

Reply:
Fixed and updated.